.changeset/kind-moose-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+"ts-essentials": patch
+---
+
+Add more IsTuple test cases
```

Comment:
can you remove this changeset as this is not a user facing change?

Reply:
yeah, will do it a moment

Reply:
Done ✅